meerk40t/core/elements.py

```diff
@@ -5737,11 +5897,26 @@ def remove_type_op(node, **kwargs):
         )
         @self.tree_operation(
             _("Delete group '%s' and all its child-elements fully") % "{name}",
-            node_type=("file", "group"),
+            node_type=("group"),
             help="",
         )
         def remove_type_grp(node, **kwargs):
+            node.remove_node()
+            self.set_emphasis(None)
+        @self.tree_conditional(lambda cond: contains_no_locked_items())
 
+        @self.tree_conditional(
+            lambda cond: len(
+                list(self.flat(selected=True, cascade=False, types=("file", "group")))
+            )
+            == 1
+        )
+        @self.tree_operation(
+            _("Delete file '%s' and all its child-elements fully") % "{name}",
```

Comment:
This may imply it deletes the file from disk. When it does not do that.

Reply:
Amendend